system/lib/embind/bind.cpp

```diff
@@ -173,6 +173,8 @@ EMSCRIPTEN_BINDINGS(builtin) {
   register_memory_view<uint16_t>("emscripten::memory_view<uint16_t>");
   register_memory_view<int32_t>("emscripten::memory_view<int32_t>");
   register_memory_view<uint32_t>("emscripten::memory_view<uint32_t>");
+  register_memory_view<int64_t>("emscripten::memory_view<int64_t>");
+  register_memory_view<uint64_t>("emscripten::memory_view<uint64_t>");
```

Comment:
What happens when these lines execute on targets that don't support BigInt64Array?

Reply:
The binding will register, but if you try to use it, it will error out. Alternatively, we could not register this when there's no bigint. Is there a preprocessor directive for this already? I tried `WASM_BIGINT`, but that' doesn't seem to exists.

Reply:
No we don't have such a thing. This seems fine.